deploy/install.sh

```diff
@@ -318,8 +318,8 @@ oc new-project tekton-pipelines || true
 
 openshift_master_default_subdomain=$(oc get ingresses.config.openshift.io cluster --output=jsonpath={.spec.domain})
 
-curl -s -L https://github.com/tektoncd/dashboard/releases/download/v0.6.1/openshift-tekton-webhooks-extension-release.yaml | sed "s/{openshift_master_default_subdomain}/${openshift_master_default_subdomain}/" | oc apply -f -
-oc apply -f https://github.com/tektoncd/dashboard/releases/download/v0.6.1/openshift-tekton-dashboard-release.yaml
+curl -s -L https://github.com/tektoncd/dashboard/releases/download/v0.6.1/openshift-tekton-webhooks-extension-release.yaml | sed "s/{openshift_master_default_subdomain}/${openshift_master_default_subdomain}/" | sed "s/openshift-pipelines/tekton-pipelines/" | oc apply -f -
```

Comment:
Hi Tim, I think the sed is a bit too generic. You do not want to update other entries with openshift-pipelines in the value entries (i.e. see: WEBHOOK_CALLBACK_URL). If you do, ok. Otherwise, making the sed more specific to namespace would be better.

Reply:
Actually that URL contains the namespace in it, that's why it needs to change too.

Reply:
Ok thanks. Will approve.